R/pkg/R/functions.R

```diff
@@ -163,3 +163,17 @@ setMethod("n", signature(x = "Column"),
           function(x) {
             count(x)
           })
+
+#' when
+#'
+#' Evaluates a list of conditions and returns one of multiple possible result expressions.
```

Comment:
We can remove the part about `otherwise` here and just make this
```
Evaluates a list of conditions and returns one of multiple possible result expressions. For unmatched expressions null is returned.
```

For the `otherwise` documentation we can say
```
If values in the specified column are null, returns the value. Can be used in conjunction with `when` to specify a default value for expressions.
```